File: GeometryPainter/Style.cs
using System.Drawing;

namespace GeometryPainter
{
    public class Style
    {
        private Color penColor ;
        private float penWidth ;
        private bool isFill ;
        private bool isSolidLine;

        public Style()
        {
            PenColor = Color.Red;
            BrushWidth = 10;
            IsFill = false;
            IsSolidLine = true;
        }

        public Style(Color penColor, float penWidth, bool isFill)
        {
            PenColor = penColor;
            BrushWidth = penWidth;
            IsFill = isFill;
            IsSolidLine = true;
        }

        public Style(Color penColor, float penWidth, bool isFill, bool isSolidLine)
        {
            this.penColor = penColor;
            this.penWidth = penWidth;
            this.isFill = isFill;
            this.isSolidLine = isSolidLine;
        }

        public Color PenColor
        {
            get { return penColor; }
            private set { penColor = value; }
        }

        public float BrushWidth
        {
            get { return penWidth; }
            private set { penWidth = value; }
        }

        public bool IsFill
        {
            get { return isFill; }
            private set { isFill = value; }
        }

        public bool IsSolidLine
        {
            get { return isSolidLine; }

            set { isSolidLine = value; }
        }
    }
}

File: TrainingWeb/js/Style.js
var Style = function  (color, penWieth, lineStyle) {
    //var arguments;
    switch (arguments.length) {
        case 0:
            this.color = "#fffffb";
            this.penWidth = 10;
            this.lineStyle = 0;
            this.isFill = false;
            break;
        case 1:
            this.color = arguments[0];
            this.penWidth = 5;
            this.lineStyle = 0;
            this.isFill = false;
            break;
        case 2:
            this.color = arguments[0];
            this.penWidth = arguments[1];
            this.lineStyle = 0;
            this.isFill = false;
            break;
        case 3:
            this.brushColor = arguments[0];
            this.lineStyle = arguments[1];
            this.isFill = arguments[2];
            break;
        default:
            alert("Input style's format is error, please re-enter！");
            break;
    }

}